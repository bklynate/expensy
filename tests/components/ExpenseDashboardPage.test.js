import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from './../../src/components/ExpenseDashboardPage';

describe('ExpenseDashboardPage Component', () => {
  it('should display both ExpenseListFilters and ExpenseList', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
