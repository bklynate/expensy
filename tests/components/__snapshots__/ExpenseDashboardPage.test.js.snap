// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseDashboardPage Component should display both ExpenseListFilters and ExpenseList 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
